### PR TITLE
Prevent saving Project Title and Project Description as blank strings

### PR DESCRIPTION
**Pull request by chiaberry**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6582

To test: https://deploy-preview-361--atd-moped-main.netlify.app/moped/projects/276/
Try changing title or description to a blank/ empty string. 

I am also adding the field configuration to the gitbook https://app.gitbook.com/@atd-dts/s/moped/dev-guides/api/configuration-files

### CODE REVIEW

File: moped-editor/src/components/DataTable/DataTable.js [hunk 1]
```diff
@@ -214,12 +219,20 @@ const DataTable = ({ fieldConfiguration, data, loading, error, refetch }) => {
...
   const executeMutation = (field = null, value = null) => {
     const mutationField = field || editField;
     const mutationValue = value !== null ? value : editValue;
+    const nullableField = fieldConfiguration.fields[mutationField].nullable ?? true;
     // Execute mutation only if there is a new value selected, prevents user
     // from attempting to save initial value, which would be null
-    if (mutationValue !== null) {
+    // Also prevents from executing migration if value has not changed
+    // And checks if field is allowed to be a blank string
+    if (mutationValue !== null && (nullableField || mutationValue !== "")) {
```
Comment (sergiogcx) [inline moped-editor/src/components/DataTable/DataTable.js:235]: Sweet! Thanks 🙏

File: moped-editor/src/views/projects/projectView/ProjectSummaryTable.js [hunk 2]
```diff
@@ -51,7 +51,9 @@ const ProjectSummaryTable = ({ data, loading, error, refetch }) => {
         editable: true,
         multiline: true,
         multilineRows: 4,
-        widthSmallAndLarger: 12
+        widthSmallAndLarger: 12,
+        nullable: false,
+        errorMessage: "Field cannot be blank",
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectSummaryTable.js:56]: Amazing! Thank you 🙏

**Review by sergiogcx — APPROVED** [review]
Thanks for patching this! 🙏

**Review by amenity — APPROVED** [review]
Works beautifully! Thanks, @chiaberry

**Pull request merged**